org.gnome.pushupmaster.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "org.gnome.pushupmaster",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "pushupmaster",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=/sys/bus/iio/devices/"
```

Comment:
this is not valid.

Reply:
it's not about this change. it's about `/sys`

Reply:
I need this to access the proximity sensor on the phone. What would be the correct way?

Reply:
Does your device expose information via https://gitlab.freedesktop.org/hadess/iio-sensor-proxy/ ?